README.md

```diff
@@ -39,44 +42,59 @@ curl -fsSL https://pkgs.zabbly.com/key.asc | sudo gpg --dearmor -o /etc/apt/keyr
 
 ### Stable repository
 
-On any of the distributions above, you can add the package repository at `/etc/apt/sources.list.d/zabbly-incus-stable.sources`:
+On any of the distributions above, you can add the package repository at
+`/etc/apt/sources.list.d/zabbly-incus-stable.sources`.
 
-```
+Run the following command to add the stable repository:
+
+```sh
+sudo bash -c 'cat <<EOF > /etc/apt/sources.list.d/zabbly-incus-stable.sources
 Enabled: yes
 Types: deb
 URIs: https://pkgs.zabbly.com/incus/stable
-Suites: DISTRO
+Suites: $(lsb_release -sc)
 Components: main
-Architectures: ARCH
-Signed-By: /etc/apt/keyrings/zabbly.asc
-```
+Architectures: $(dpkg --print-architecture)
+Signed-By: /etc/apt/keyrings/zabbly.gpg
 
-Make sure to replace `DISTRO` with one of `focal`, `jammy`, `bullseye` or `bookworm`
-and then replace `ARCH` with one of `amd64` or `arm64`.
+EOF'
+```
 
 ### Daily repository
 
-On any of the distributions above, you can add the package repository at `/etc/apt/sources.list.d/zabbly-incus-daily.sources`:
+On any of the distributions above, you can add the package repository at
+`/etc/apt/sources.list.d/zabbly-incus-daily.sources`.
 
-```
+Run the following command to add the daily repository:
+
+```sh
+sudo bash -c 'cat <<EOF > /etc/apt/sources.list.d/zabbly-incus-stable.sources
 Enabled: yes
 Types: deb
-URIs: https://pkgs.zabbly.com/incus/daily
-Suites: DISTRO
+URIs: https://pkgs.zabbly.com/incus/stable
+Suites: $(lsb_release -sc)
 Components: main
-Architectures: ARCH
-Signed-By: /etc/apt/keyrings/zabbly.asc
-```
+Architectures: $(dpkg --print-architecture)
+Signed-By: /etc/apt/keyrings/zabbly.gpg
 
-Make sure to replace `DISTRO` with one of `focal`, `jammy`, `bullseye` or `bookworm`
-and then replace `ARCH` with one of `amd64` or `arm64`.
+EOF'
+```
 
 ### Installing Incus
 
-Update your repository list with: `apt-get update`
+Update your repository list with:
+
+```sh
+sudo apt-get update
+```
+
+Then to install Incus, run:
 
-Then to install Incus, run: `apt-get install incus`
+```sh
+sudo apt-get install incus
+```
```

Comment:
`sudo` isn't part of the standard Debian install which is why I didn't make use of it initially. It may be best to just point out that all instructions need to be performed as root instead.

Reply:
Prefacing commands with `#` to indicate a root prompt might be an idea, it means the commands can't be copy-pasted directly but for me that's a good thing.